bigtop-packages/src/common/spark/do-component-build

```diff
@@ -30,6 +30,22 @@ BUILD_OPTS="-Divy.home=${HOME}/.ivy2 -Dsbt.ivy.home=${HOME}/.ivy2 -Duser.home=${
             -Dguava.version=27.0-jre \
             $SPARK_BUILD_OPTS"
 
+#add pandoc by wget and install in openEuler
+. /etc/os-release
+OS="$ID"
+
+if [ "${OS}" = "openEuler" ]; then
+  if [ $HOSTTYPE = "aarch64" ]; then
+    wget https://github.com/jgm/pandoc/releases/download/2.19.2/pandoc-2.19.2-linux-arm64.tar.gz
+    tar -xf pandoc-2.19.2-linux-arm64.tar.gz
+    cp -r pandoc-2.19.2/bin/pandoc /usr/bin/
+  else
+    wget https://github.com/jgm/pandoc/releases/download/2.19.2/pandoc-2.19.2-linux-amd64.tar.gz
+    tar -xf pandoc-2.19.2-linux-amd64.tar.gz
+    cp -r pandoc-2.19.2/bin/pandoc /usr/bin/
+  fi
+fi
+
```

Comment:
Please also move `pandoc` into `bigtop_toolchain`.
https://github.com/apache/bigtop/blob/master/bigtop_toolchain/manifests/renv.pp#L25

Reply:
We install R and Pandoc in Bigtop_toolchain.
Please do not install the dependency packages when building components.
@MacChen01

Reply:
Pandoc installation moved into bigtop_toolchain. @guyuqi 
commits: https://github.com/apache/bigtop/pull/1148/commits/6d5fa5ad7ff65d03b9836f074b2129e5fb5a6c5d